recipes/minian/meta.yaml

```diff
@@ -0,0 +1,126 @@
+# Note: there are many handy hints in comments in this example -- remove them when you've finalized your recipe
+
+# Jinja variables help maintain the recipe as you'll update the version only here.
+# Using the name variable with the URL in line 14 is convenient
+# when copying and pasting from another recipe, but not really needed.
+{% set name = "MiniAn" %}
+{% set version = "0.2.5" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://test-files.pythonhosted.org/packages/3d/3d/21520cc69ba85a4f48b2a0baa631f2473f9173f6af3f6600a0763cc97a8b/MiniAn-0.2.5.tar.gz
```

Comment:
I don't see a formal pypi.org page for this package, so I guess this is ok. If the package does go up on pypi.org in the future, then consider changing this to the canonical form:

https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L14

Reply:
yes we know this, for now it's okay, we will change it to the canonical form when we are on pypi